### DISCUSSION TITLE
[hidden] attribute doesn't support "until-found"

### DISCUSSION THREAD

**Discussion by IHIutch** [alpinejs/alpine#4086] [3. Feature ideas]
`[hidden]` is no longer a boolean attribute. Trying to use a non-boolean value such as 'until-found' does not work as expected

[The hidden attribute spec](https://html.spec.whatwg.org/multipage/interaction.html#the-hidden-attribute)

**Comment by SimoTod** [comment]
Fixed by https://github.com/alpinejs/alpine/pull/4099
